app/controllers/transfer_controller.rb

```diff
@@ -46,11 +46,13 @@ def files
       thesis.files.attach(file.blob)
       flash[:success] += "#{file.filename}<br>".html_safe
     end
+    transfer.save # triggers update to files_counts
     redirect_to transfer_path(transfer.id, view_all: params[:view_all] || 'false')
   end
 
   def select
-    @transfer = Transfer.all
+    @transfers = Transfer.all.with_attached_files
+    @transfers = @transfers.includes(:user).includes(:department)
```

Comment:
I don't recall if this was split for weird functional reasons or because it fit my brain at the time. You may notice that the old value wasn't using the correct instance variable and thus nothing that happened here affected the view at all which was interesting as well.

I'll confirm if we can chain these or not and either update the code or report back (on paper they look chainable for sure).

Reply:
I'm going to move this to a single line chain. Our tests and a manual reconfirmation it still works as expected locally confirm Rails treats this as the same query whether it is a single or split across multiple lines.

Reply:
Is there another commit addressing that? Sorry if I'm missing something.

Reply:
Okay, it's there now for real this time!